index.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module "moleculer-apollo-server" {
+	import { ServiceSchema } from "moleculer";
+	import { Config } from "apollo-server-core";
+	export {
+		GraphQLUpload,
+		GraphQLExtension,
+		gql,
+		ApolloError,
+		toApolloError,
+		SyntaxError,
+		ValidationError,
+		AuthenticationError,
+		ForbiddenError,
+		UserInputError,
+		defaultPlaygroundOptions,
+	} from "apollo-server-core";
+
+	export * from "graphql-tools";
+
+	export interface ApolloServerOptions {
+		path: string;
+		disableHealthCheck: boolean;
+		onHealthCheck: () => {};
+	}
+
+	export class ApolloServer {
+		createGraphQLServerOptions(req: any, res: any): Promise<any>;
+		createHandler(options: ApolloServerOptions): void;
+		supportsUploads(): boolean;
+		supportsSubscriptions(): boolean;
+	}
+
+	export interface GraphQLNodeResolver {
```

Comment:
This type accepts additional parameters of `nullIfError` and `params`.

I think this interface name is a bit misleading since it feels like its a "GraphQL" type rather rather than a moleculer-apollo-server type.  How about `ActionResolverSchema` or `ActionResolverDefinition` since this object is used by a function called `createActionResolver`?

Reply:
I think `ActionResolverSchema` is the good one.